tests/gamma/csm_observability_test.py

```diff
@@ -195,7 +196,15 @@ def test_csm_observability(self):
                 "Letting test client run for %d seconds to produce metric data",
                 TEST_RUN_SECS,
             )
-            time.sleep(TEST_RUN_SECS)
+            for i in range(0, TEST_RUN_SECS // 10):
+                time.sleep(10)
+                logger.info(
+                    requests.get(
```

Comment:
todo: wrap the requests call in a try block to catch requests.exceptions.RequestException - https://requests.readthedocs.io/en/latest/user/quickstart/#errors-and-exceptions

Reply:
Done